res/graphics/background_fully_transparent.svg

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Again, is this file still needed?

Reply:
yes, see https://github.com/streetcomplete/StreetComplete/pull/4642#discussion_r1104892790